doc/integrator/docker.rst

```diff
@@ -80,3 +80,53 @@ You can also remove the unused named images, that should be done manually:
 
 * List the images with ``docker images``.
 * Remove the unwanted images with ``docker rmi <image>:<tag>``.
+
+
+Configure project for OpenShift
+-------------------------------
+
+This section is specific to projects using OpenShift for deployment, relying
+on Travis to build Docker images that are then pushed to Docker Hub.
+
+Edit file `.travis.yml` and update the environment variables:
+
+.. prompt:: bash
+
+    env:
+      global:
+        - PROJECT=<package name>
+        - OPENSHIFT_PROJECT=gs-gmf-<package name>
+        - HELM_RELEASE_NAMES=int,prod
+        - OPENSHIFT_URL=https://api.openshift-ch-1.camptocamp.com/
+        - DOCKER_USERNAME=<FIXME>
+        - secure: <encrypted password>
+
+The encrypted password is obtained using the ``travis encrypt`` command from
+within the project directory:
+
+.. prompt:: bash
+
+    cd <project directory>
+    gopass gs/ci/dockerhub/password
```

Comment:
Perhaps a doc about gopass would be useful somewhere?

Reply:
OK, but I don't remember what you did and have no link :P

Reply:
https://github.com/camptocamp/geospatial-ci-pass/blob/master/README.md :-)